### DISCUSSION TITLE
[Feature Request] Generate CSS button on Admin bar

### DISCUSSION THREAD

**Discussion by cuongthachHQ** [wind-press/windpress#9] [General]
Right now, I must generate CSS (purge CSS) via **WindPress** > **Setting** > **Generate** to purge CSS. It's inconvenient, and a better solution would be to turn on **Enable WindPress panel on the front page** to open the widget on the preview page for purging.
It's truly not good.

Why not add a Generate button on the Admin bar like other cache plugins do?

Backend or frontend doesn't matter, just click on the Button on admin bar and it's done.

Or you have an option auto generate when save the page only?

**Comment by cuongthachHQ** [comment]
@suabahasa

**Comment by suasgn** [comment]
Hi, caching requires you to load the WindPress Engine into your browser. This may slow down your site if it loads in every place where the admin bar is loaded.

**Comment by suasgn** [comment]
We have some feature for some integrations that allow to generate the cache on the save
